### DISCUSSION TITLE
Using /public instead of / (root)

### DISCUSSION THREAD

**Discussion by ProgrammingParadox** [community/community#23520] [Other Feature Feedback, Questions, & Ideas]
<p>In Github Pages, how can I make it use the /public folder instead of the / (root)? I see the /doc option but nothing else. (As in, how can I make an express app appear as a github page)</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>maybe you mean branches ? like this ?</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181077508-554d5dfc-1200-4c8c-a307-93f3c43417b0.png" alt="image" data-base62-sha1="aL2FqxuxAkB37QNkoCvWrPUGL2F" width="496" height="430"></p>
<p>but for either <code>root</code> or <code>docs</code>, they are the only two choices even if you add another folder</p>
<p>you can check this out too</p>
<aside class="quote" data-post="1" data-topic="10237">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/can-i-define-a-custom-source-or-folder-from-which-my-site-hosted-on-github-pages-can-load-from/10237">Can I define a custom source or folder from which my site hosted on GitHub pages can load from?</a> <a class="badge-wrapper  bullet" href="/c/github-help/github-pages/31"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #c8e1ff;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Questions and discussion about creating a website directly from a repository on GitHub.com.">GitHub Pages</span></a>
  </div>
  <blockquote>
    Instead of the normal options as the source for my site deployed with pages, that is the master branch or the docs folder in the master branch, can I define another source, like a folder called site or pages or some custom name. 
This will be really helpful, as I really like using specific names for my repo folders.
  </blockquote>
</aside>
